bittensor/_neuron/__init__.py

```diff
@@ -20,18 +20,18 @@
 version_split = __version__.split(".")
 __version_as_int__ = (100 * int(version_split[0])) + (10 * int(version_split[1])) + (1 * int(version_split[2]))
 
-from .text import template_miner, template_server, advanced_server, core_validator, multitron_server
-
-
+from .text import template_miner, template_server, advanced_server, core_validator, multitron_server, core_server
 __all_neurons__ =  { 'text_template_miner': template_miner.neuron, 
                      'text_core_validator': core_validator.neuron,
                      'text_template_server':template_server.neuron,
                      'text_advanced_server':advanced_server.neuron,
-                     'multitron_server': multitron_server.neuron}
+                     'multitron_server': multitron_server.neuron,
```

Comment:
Does the mulitron server still work?

Reply:
it should not work, and it is also not hard to make it work, 
but also it is not a priority to renew it and validate its performance, 
so lets depricate it for now